### PR TITLE
AM-4834: Fix minor typos in tutorials doc

### PR DESCRIPTION
**Pull request by uma-kt**
Fixed minor typos in the tutorials:
- BookInfo
- Iperf

### CODE REVIEW

File: versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx [hunk 1]
```diff
@@ -446,7 +446,7 @@ Perform these steps:
    serviceexport.networking.kubeslice.io/reviews created
    ```
 :::info
-This completes the deployment of Bookinfo Appication on a Slice.**
+This completes the deployment of Bookinfo Application on a Slice.**
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx:449]: Application can be `application` and Slice can be a `slice` as it is a paragraph (note).

File: versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application.mdx [hunk 2]
```diff
@@ -24,7 +24,7 @@ see [Installing Istio](/versioned_docs/version-0.3.0/getting-started-with-cloud-
 - You have the slice created across worker clusters. For more information, 
 see [Creating a Slice](/versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/installing-kubeslice/creating-a-slice.mdx).
 
-## Creating the Bookinfo Deployment YAML Files
+## Creating the BookInfo Deployment YAML Files
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application.mdx:27]: As per https://istio.io/latest/docs/examples/bookinfo/, it is `Bookinfo`

**Review by veenamj-avesha — COMMENTED** [review]
Provided feedback. Please incorporate them as required.

File: versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application.mdx (hunk 2 above)
Comment (uma-kt) [reply versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application.mdx:27]: updated

File: versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx [hunk 3]
```diff
@@ -446,7 +446,7 @@ Perform these steps:
    serviceexport.networking.kubeslice.io/reviews created
    ```
 :::info
-This completes the deployment of Bookinfo Appication on a Slice.**
+This completes the deployment of Bookinfo Application on a Slice.**
```
[the remark this answers is not in the document]
Comment (uma-kt) [reply versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx:449]: updated

File: versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx [hunk 4]
```diff
@@ -19,7 +19,7 @@ see [Installing KubeSlice](/versioned_docs/version-0.3.0/getting-started-with-cl
 To install the BookInfo Application on a single cluster, you must create a slice without Istio. For more information, 
 see [Creating a Slice](/versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/installing-kubeslice/creating-a-slice.mdx).
 
-## Creating the Bookinfo Deployment YAML Files
+## Creating the BookInfo Deployment YAML Files
```
[the remark this answers is not in the document]
Comment (uma-kt) [reply versioned_docs/version-0.3.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx:22]: updated

File: versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx (hunk 1 above)
Comment (uma-kt) [reply versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/tutorials/deploying-the-bookinfo-application-on-an-intra-cluster-slice.mdx:449]: updated

**Review by veenamj-avesha — APPROVED** [review]
Changes LGTM

**Pull request merged**